Mathlib/Data/Multiset/Functor.lean

```diff
@@ -0,0 +1,147 @@
+/-
+Copyright (c) 2018 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Johannes Hölzl, Simon Hudon, Kenny Lau
+
+! This file was ported from Lean 3 source module data.multiset.functor
+! leanprover-community/mathlib commit 1f0096e6caa61e9c849ec2adbd227e960e9dff58
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Data.Multiset.Bind
+import Mathlib.Control.Traversable.Lemmas
+import Mathlib.Control.Traversable.Instances
+
+/-!
+# Functoriality of `Multiset`.
+-/
+
+
+universe u
+
+namespace Multiset
+
+open List
+
+instance : Functor Multiset where map := @map
```

Comment:
```suggestion
instance functor : Functor Multiset where map := @map
```
It would be good to name the instance explicitly since the name is used later in the file.